big_tests/tests/inbox_SUITE.erl

```diff
@@ -413,6 +420,14 @@ msg_sent_stored_in_inbox(Config) ->
         check_inbox(Bob, BobConvs)
       end).
 
+msg_sent_stored_in_inbox_queryid(Config) ->
+    escalus:fresh_story(Config, [{alice, 1}, {bob, 1}], fun(Alice, Bob) ->
+        #{ Alice := AliceConvs, Bob := BobConvs } = given_conversations_between(Alice, [Bob]),
+        %% Both check inbox
+        check_inbox(Alice, AliceConvs, <<"Query1">>),
+        check_inbox(Bob, BobConvs, <<"Query2">>)
+      end).
+
```

Comment:
Considering that this test and the previous are so similar except for the value of that query-id, I'd refactor them into a single helper function, something like 

```erlang
msg_sent_stored_in_inbox(Config) ->
    test_msg_sent_stored_in_inbox(Config, undefined).

msg_sent_stored_in_inbox_queryid(Config) ->
    test_msg_sent_stored_in_inbox(Config, query_id).

test_msg_sent_stored_in_inbox(Config, QueryId) ->
    escalus:fresh_story(Config, [{alice, 1}, {bob, 1}], fun(Alice, Bob) ->
        #{ Alice := AliceConvs, Bob := BobConvs } = given_conversations_between(Alice, [Bob]),
        %% Both check inbox
        check_inbox(Alice, AliceConvs, maybe_make_query_id(QueryId)),
        check_inbox(Bob, BobConvs, maybe_make_query_id(QueryId))
      end).

maybe_make_query_id(undefined) -> undefined;
maybe_make_query_id(query_id) -> create_some_unique_binary();
```

Note all function names and how I chose helper functions are again just me writing fast and thinking little, so take these things as ideas to consider and work around, not as exact requirements 🙂